alignak/objects/realm.py

```diff
@@ -79,7 +79,7 @@ class Realm(Itemgroup):
         'realm_name':
             StringProp(fill_brok=['full_status']),
         'alias':
-            StringProp(fill_brok=['full_status']),
+            StringProp(default=''),
```

Comment:
Previously, the alias field did not even existed in the object ... it has been added fo have a certain compatibility with the other objects.